src/chi/components/picker/picker-groups.scss

```diff
@@ -0,0 +1,34 @@
+@import 'horizontal-picker-groups';
+
+.m-pickerGroup {
+  &.-fluid {
+    display: flex;
+    input {
+      &[type='checkbox'],
+      &[type='radio'] {
+        &.a-picker {
+          & + label {
+            display: flex;
+            flex: 1 0 0;
+
+            &.-notFluid {
+              flex: 0 1 0;
+            }
+
+            &.-align--left {
+              justify-content: flex-start;
+            }
+
+            &.-align--center {
```

Comment:
@jllr Fluid picker group labels should be centered by default to match button group behavior. Can you fix and we can get this merged?